rest/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>mushroom-hunter-portal</artifactId>
+        <groupId>cz.fi.muni.pa165</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>rest</artifactId>
+
+
+    <dependencies>
+
+        <dependency>
+            <groupId>cz.fi.muni.pa165</groupId>
+            <artifactId>api</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
```

Comment:
ja bych to tam nechal aby to bylo explicitne jasny/ teoreticky muzes pouzit jinou service nebo ji nemusis pouzit vubec